lib/pbench/server/api/resources/alembic.ini

```diff
@@ -0,0 +1,85 @@
+# A generic, single database configuration.
```

Comment:
I was just glancing through here to see where you defined your postgres objects because I think I'm adding some, too; and scanning through I had a thought about organization.

My original intent with the `resources` subdirectory was to hold "API resources", as in, URI endpoints. You're bundling in stuff like the DB model files and alembic framework that, in my original conception, don't belong, and makes that level of the class tree "muddy".

I never really even noticed that when I was concentrating on reviewing the content of the files... but it stands out to me now.

I think I'd like to see a structure maybe more like:

```
+ lib/pbench/server/
--+ database
----+ upgrade
----+ models
--+ api
----+ resources
```

I'm not entirely sure where the authorization modules goes under this schema. Maybe it's a "util" or maybe there's a separate "users" subtree to hold authorization along with metadata management and such.

Anyway, I think that "server", and the "user" and "database" concepts, extend beyond just the "API", and I'd like to start thinking about laying the foundation as we extend the model.

Reply:
Yes I agree, I database models, alembic files, auth modules do not belong to the resources folder, I just added them to resources as an initial place holder. I had this in my mind for a while but now that you mentioned I think I should put some efforts in segregating them, that would be good